### DISCUSSION TITLE
Successful build: S2-WL-ST + S5-EH1P5K-L

### DISCUSSION THREAD

**Discussion by ThomasPrior** [alienatedsec/solis-ha-modbus-cloud#22] [Show and tell]
Just wanted to say thank you for this project. Successfully built today and I have Solis Cloud with remote control working and HomeAssistant connected via Modbus over TCP.

Hardest part for me was my poor soldering skills!

I'd like to swap the TP-Link switch for a UniFi managed switch in future, then I can segregate my datalogger from my trusted devices.

<img width="1280" alt="image" src="https://github.com/alienatedsec/solis-ha-modbus-cloud/assets/34111848/193f4183-655e-4108-b7f7-9025bcce9284">

![image](https://github.com/alienatedsec/solis-ha-modbus-cloud/assets/34111848/ed3ac0e6-2fa0-4fcc-b8cc-e82d3a6dd97a)

**Comment by alienatedsec** [comment]
Thanks for sharing your experience. I hope the minimised enclosure was not too much trouble. 

I have seen another din enclosure which could be better for this project 

15 Way Consumer Unit, Distribution Protection Box, IP65 Electrical Enclosure, Waterproof Transparent Window Consumer Unit, Consumer Circuit Breaker Box https://amzn.eu/d/4HqGb4s
